include/ofi_shm.h

```diff
@@ -68,7 +69,7 @@ extern "C" {
 
 #define SMR_FLAG_IPC_SOCK (1 << 2)
 
-#define SMR_CMD_SIZE		128	/* align with 64-byte cache line */
+#define SMR_CMD_SIZE		256	/* align with 64-byte cache line */
```

Comment:
This change is due to extended `ipc_info` struct, after discussing with @aingerson

Reply:
Doesn't this break the SMR protocol, needing a version bump?

Reply:
Bumped `SMR_VERSION` to 3.